src/System.Net.Sockets.Legacy/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -1384,7 +1388,7 @@ public void ReceiveFromV6BoundToSpecificV4_NotReceived()
         }
 
         [Fact]
-        [PlatformSpecific(~PlatformID.Linux)]
+        [PlatformSpecific(~(PlatformID.Linux | PlatformID.OSX))]
```

Comment:
I can imagine that other platforms will support this.  I thought about doing a "conditional" exclusion based on whether ReceiveFrom actually throws PNSE - but that seemed oddly circular to me. :)  Also, it would mean these show up as "skipped" tests at runtime, rather than simply not being discovered in the first place.